index.js

```diff
@@ -283,22 +289,30 @@ function showDataFile(e, o){
 
     text = e.target.result
     text = text.split("/Game/Campaign_Main/Quest_Campaign_Ward13.Quest_Campaign_Ward13")[0]
-    text = text.split("/Game/Campaign_Main/Quest_Campaign_City.Quest_Campaign_City")[1].replace(/Game/g,"\n")
+    main_campaign = text.split("/Game/Campaign_Main/Quest_Campaign_City.Quest_Campaign_City")[1]
+    if (main_campaign) {
+        text = main_campaign.replace(/Game/g,"\n")
+    } else { // subject 2923 campaign
+        text = text.split("/Game/Campaign_Clementine/Quests/WardPrime/Quest_WardPrime_Template.Quest_WardPrime_Template")[0]
+        text = text.split("/Game/World_Rural/Templates/Template_Rural_Overworld_02.Template_Rural_Overworld_02")[1].replace(/Game/g,"\n")
+    }
 
     textArray = text.split("\n")
 
-   adText = e.target.result
-
-   adText = adText.split("\n")
-   tempList = []
-   for(i = 0; i < adText.length; i++)
-   {
-     if (String(adText[i]).includes('Adventure') === true)
-     {
-       tempList.push(adText[i])
-     }
-   }
-   adText = tempList[1]
+    adText = e.target.result
+
+    adText = adText.split("\n")
+    tempList = []
+    for(i = 0; i < adText.length; i++)
+    {
+        if (String(adText[i]).includes('Adventure') === true)
+        {
+            tempList.push(adText[i])
+        }
+    }
+    // regardless of campaign type, the last line collected will have our current adventure data
+    adText = tempList[tempList.length - 1]

```

Comment:
This is the big new line here, the rest of this block is just adjusting spacing because it had been off.

As detailed in #9 the place in the file that it was grabbing adventure mode data from may have been fine for save files with a From the Ashes campaign, but didn't work for save files with a Subject 2923 campaign. There are multiple places in the file that have this data, so I moved to another spot towards the end of the file where it appears in both campaign types.